src/carbonio-files-ui-common/utils/ActionsFactory.ts

```diff
@@ -344,7 +344,7 @@ export function canDownload({ nodes }: { nodes: NodeDownload[] }): boolean {
 		throw Error('cannot evaluate canDownload on empty nodes array');
 	}
 	const node = nodes[0];
-	return nodes.length === 1 && isFile(node) && node.rootId !== ROOTS.TRASH;
+	return node.rootId !== ROOTS.TRASH;
```

Comment:
now that we are checking more than 1 node we need to check that every node satisfy the requirement !== ROOTS.TRASH.